### PR TITLE
fix(profile): update profile image

### LINKED ISSUE #270: In the Supplier user you should update the header data.
### Actual Result
When updating the Supplier profile, the company logo image in the header does not change.
chrome_pTnWK2FLij

### Expected
When updating the supplier profile to immediately change the header logo

#### Extra task
Logo size changes in different views
chrome_aSE1gvzeOu

### PR DESCRIPTION
**Pull request by eduardoquispe**
### Description

Please include a summary of the change.

fixes # (issue)

#### Screenshots

If applicable, add screenshots about your changes

#### Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How to test it

If applicable, describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

### Checklist:

- [ ] Doesn't break the current code.
- [ ] Passes linters and test, also is building.
- [ ] Doesn't have spelling or grammatical problems.
- [ ] Doesn't have unnecessary comments or debugging code.
- [ ] The branch is updated with the last dev branch changes.

### CODE REVIEW

**Review by e-azocar — APPROVED** [review]

File: controllers/providers.ts [hunk 1]
```diff
@@ -126,7 +126,7 @@ export const updateProvider = async (req: NextApiRequest, res: NextApiResponse)
             mutation: UPDATE_PROVIDER,
             variables: { ...fields, id },
           })
-          return res.status(204).json({
+          return res.status(200).json({
```
Comment (Israel-Laguan) [inline controllers/providers.ts:129]: Maybe 205 would be better, as you notice when the data changes we can reset UI

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/205

File: controllers/providers.ts [hunk 2]
```diff
@@ -138,7 +138,7 @@ export const updateProvider = async (req: NextApiRequest, res: NextApiResponse)
           mutation: UPDATE_PROVIDER,
           variables: { ...fields, logoUrl, id },
         })
-        return res.status(204).json({
+        return res.status(200).json({
```
Comment (Israel-Laguan) [inline controllers/providers.ts:141]: Maybe 205 would be better, as you notice when the data changes we can reset UI

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/205

File: controllers/providers.ts [hunk 3]
```diff
@@ -148,7 +148,7 @@ export const updateProvider = async (req: NextApiRequest, res: NextApiResponse)
         mutation: UPDATE_PROVIDER,
         variables: { ...fields, logoUrl: '' },
       })
-      res.status(204).json({
+      res.status(200).json({
```
Comment (Israel-Laguan) [inline controllers/providers.ts:151]: Maybe 205 would be better, as you notice when the data changes we can reset UI

Ref: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/205

File: controllers/providers.ts (hunk 1 above)
Comment (eduardoquispe) [reply controllers/providers.ts:129]: @Israel-Laguan  With 204 or 205 it does not return me the json to be able to do the update in the session storage, I could put 204 or 205 but I would have to call another endpoint to return the updated data of the profile, instead with 200 I can do the update with the response.

Comment (Israel-Laguan) [reply controllers/providers.ts:129]: That shouldn't be a problem

@Eghinner @e-azocar please help him

**Comment by angelik0828** [comment]
## ✔️ It has been validated when adding an image, it is updated, and it updates the image on the top right with the new image.
 
 image

**Review by angelik0828 — APPROVED** [review]

**Comment by Eghinner** [comment]
@eduardoquispe Could you solve the extra task?

**Comment by eduardoquispe** [comment]
@Eghinner yes, but it would be at night

**Comment by Eghinner** [comment]
Update branch

File: controllers/providers.ts (hunk 1 above)
Comment (e-azocar) [reply controllers/providers.ts:129]: Yes, 204 and 205 are no content HTTP status, so, you can't send a json or text

**Comment by eduardoquispe** [comment]
@Eghinner  updated with logo size

**Review by Israel-Laguan — APPROVED** [review]

**Review by Eghinner — APPROVED** [review]

**Pull request merged**